### DISCUSSION TITLE
English as base language, german abstract not formatting ß correctly

### DISCUSSION THREAD

**Discussion by Tyde** [tudace/tuda_latex_templates#438] [Q&A] [answered]
Hi, I feel a bit dumb for asking this but I just cannot get it to work. I've written my thesis in english but will have to provide a german abstract next to my english one. Therefore, my setup looks like this (Using LuaLaTeX):

```latex
\documentclass[
	english,

...
...
\usepackage[T1]{fontenc}
\usepackage[english,ngerman]{babel}


\begin{abstract}
	English abstract

\end{abstract}

\selectlanguage{ngerman}
\begin{abstract}[ngerman]
    German abstract with ß
\end{abstract}
\selectlanguage{english}
```

But all ß are printed out as SS, while the other umlaute do work however. What am I missing?

**Comment by TeXhackse** [comment, marked as answer]
Remove the fontenc package. One should not load a T1 encoding with lualatex.

**Reply by Tyde** [reply]
Thanks. The line wasn't in there originally and I just tried out stuff out of desperation. Sadly removing it doesn't help the issue.

**Reply by TeXhackse** [reply]
Then you probably load some package which loads that indirectly. Minimal example should be fine there. 

full MWE: 

```
\documentclass[english]{tudapub}

\usepackage[main=english,ngerman]{babel}
\begin{document}

\begin{abstract}
	English abstract

\end{abstract}

\begin{abstract}[ngerman]
    German abstract with ß
\end{abstract}

content...
\end{document}
```

**Reply by Tyde** [reply]
You're correct. It seems that at some point I added `\usepackage[charter]{mathdesign}` to my preamble. Removing that worked.
